contracts/Lens_SpokePool.sol

```diff
@@ -9,10 +9,25 @@ import "./ZkSync_SpokePool.sol";
  * @custom:security-contact bugs@across.to
  */
 contract Lens_SpokePool is ZkSync_SpokePool {
-    /// @custom:oz-upgrades-unsafe-allow constructor
+    address public immutable usdcAddress;
```

Comment:
How should we handle if this contract ever extends the CCTP_Adapter, which has its own usdcAddress?

Reply:
`usdcAddress` is upgradeable USDC, so we should retain the same USDC address as part of a CCTP upgrade.